docs/tut-making-jettycat.md

```diff
@@ -389,17 +396,17 @@ Here we compare a copy's horizontal coordinate to a camera's left side. We also
 
 Let's add a template for star bonuses that will increment score when collected. We will do the following:
 
-1. Set up a score variable in our room's "On Create" code.
+1. Set up a score variable in our room's "Creation" code.
 2. Create a new template for star bonuses.
-3. Add a bit of logic to the star's "On Step" event that will destroy the star when collided with the cat.
+3. Add a bit of logic to the star's "Frame start" event that will destroy the star when colliding with the cat.
```

Comment:
This can *(and should)* be replaced with a "collision with a template" event from ct.place